src/test/js/multibranch/noRunsActivityPage.js

```diff
@@ -1,24 +1,61 @@
 const jobName = 'noRunsActivityPage';
+const tmp = require('tmp');
+
+const repo = tmp.dirSync();
+const pathToRepo = repo.name;
+const path = require("path");
+const sourceRep = './src/test/resources/multibranch/test_results';
+const git = require("../../../main/js/api/git");
+
+
 /** @module failing
  * @memberof testcases
  * @description TEST: basic tests around the failing pipeline. Test whether the result is not collapsed.
  */
 module.exports = {
+    // ** creating a git repo */
+    before: (browser, done) => {
+          // we creating a git repo in target based on the src repo (see above)
+          git.createRepo(sourceRep, pathToRepo).then(done);
+    },
     /**
      * Create Multibranch Pipeline Job "noRunsActivityPage" with no branches.
      * @param browser
      */
-    'Step 01': function (browser) {
+    'Create job': (browser) => {
```

Comment:
I disagree- I use this a lot and find it much more helpful to have a meaningful description> That is why ti is a string.